# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../entrance/edjecc_is_not_cc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./data/themes/taillights/Makefile.am.orig	2005-03-01 15:54:36.026396912 -0300
+++ ./data/themes/taillights/Makefile.am	2005-03-01 15:55:03.271255064 -0300
@@ -1,7 +1,7 @@
 EXTRA_DIST= taillights.edc
 SUBDIRS = parts programs fonts images
 DB = taillights.eet
-CC = edje_cc
+ECC = edje_cc
 INCLUDES = -v -id ./images -fd ./fonts
 
 CLEANFILES = $(DB)
@@ -12,4 +12,4 @@
 all: $(DB)
 
 $(DB): taillights.edc
-	$(CC) $(INCLUDES) taillights.edc $@
+	$(ECC) $(INCLUDES) taillights.edc $@
